spaces/arckyf/skin.scss

```
@@ -0,0 +1,9 @@
+.balancer {
```

The key of your space must also but the folder name and skin class name.
```suggestion
.arckyf {
```